<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyDetailsWidget</class>
 <widget class="QWidget" name="CompanyDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Company Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="phoneEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="addressLineEdit1"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="addressLineEdit2"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="addressLineEdit3"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="addressLineEdit4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
